config/settings/base.py

```diff
@@ -397,3 +398,7 @@
 FEATURE_FEEDBACK_MECHANISM_ENABLED = strtobool(
     os.getenv("FEATURE_FEEDBACK_MECHANISM_ENABLED", "False")
 )
+
+SESSION_ENGINE = "django.contrib.sessions.backends.cached_db"
```

Comment:
As outlined in [the docs](https://docs.djangoproject.com/en/4.2/topics/http/sessions/), this changes the SESSION_ENGINE for all of Django's session usage. Is that what you want?